scripts/extensions/pool_update.precheck

```diff
@@ -49,10 +48,10 @@ def parse_control_package(session, yum_url):
     return items[0].getAttribute('control')
 
 
-def execute_precheck(session, control_package, yum_conf_file, nogpgcheck):
+def execute_precheck(session, control_package, yum_conf_file):
     FNULL = open(os.devnull, 'w')
     retcode = subprocess.call('yum -c ' + yum_conf_file + ' install -y ' +
```

Comment:
Just a general comment: I think it is a bad idea to create a string that is interpreted by a shell unless this is absolutely necessary and here it doesn't look like this is the case. It is much safer to provide arguments as an array of strings that are not passed through a shell. See: https://docs.python.org/2/library/subprocess.html#using-the-subprocess-module